src/enzyme_ad/jax/Passes/EnzymeHLOOpt.cpp

```diff
@@ -15106,6 +15199,8 @@ struct EnzymeHLOOptPass
     RewritePatternSet patterns(context);
     mlir::enzyme::populateWithGenerated(patterns);
 
+    patterns.add<SliceExtend>(context); // Add the new SliceExtend pattern
```

Comment:
I'm not sure where I should add this pattern.

Reply:
that's fine for now, just make sure its in transform

Reply:
It is in TransformOps.td